src/routes/CommunitySettings/ImportExportSettingsTab/ImportExportSettingsTab.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import Button from 'components/Button'
+import './ImportExportSettingsTab.scss'
+
+export default class ImportExportSettingsTab extends Component {
+  handleClickUpload = () => {
+    const { upload } = this.props
+    upload()
+  }
+
+  render () {
+    const { community } = this.props
+    const { name } = community
+
+    return <div>
+      <div styleName='header'>
+        <div styleName='title'>Import Posts by CSV</div>
+      </div>
+      <div styleName='help'>
+        You can select a CSV file to import posts into {name}. Posts will be created by you. The file must have columns with the following headers:
+        <ul>
+          <li>title: text</li>
+          <li>description: text</li>
+          <li>location: text</li>
+          <li>type: one of discussion, request, offer, resource, event, project</li>
+          <li>start_date (optional): e.g. 20200730-12:23:12.000+00 (other date formats may work)</li>
+          <li>end_date (optional): e.g. 20200731-12:23:12.000+00 (other date formats may work)</li>
+          <li>image_urls: 1 or more image URLs separated by spaces and/or commas</li>
+          <li>topics: up to 3 topic names separated by spaces and/or commas e.g. “food organic”</li>
+          <li>is_public: true or false</li>
+        </ul>
+      </div>
+      <div styleName='button-wrapper'>
```

Comment:
It'd be a nice to have to report back posts that didn't import due to an error.

Reply:
yeah definitely, but thats a pretty significant project i decided to hold off on and created a ticket for https://github.com/Hylozoic/hylo-evo/issues/601